src/components/widgets/sales-cycle/sales-cycle.directive.coffee

```diff
@@ -23,15 +23,20 @@ module.controller('WidgetSalesCycleCtrl', ($scope, $q, ChartFormatterSvc, $filte
   # --------------------------------------
   w.initContext = ->
     if $scope.isDataFound = angular.isDefined(w.content) && !_.isEmpty(w.content.status_average_durations)
+
+      meta = if w.metadata.status_selection.reach == 'dashboard' then ImpacDashboardsSvc.getCurrentDashboard().status_selection else w.metadata.status_selection
```

Comment:
important: **test here is not ok** - should not be testing `widget.metadata.status_selection.reach`: what about a newly added widget that wouldn't have any metadata defined?

Please follow logic described in previous comment (should be defaulted to the widget.metadata if defined, or to the dashboard.medata if defined).

Same thing for other widgets